ansible/roles/airflow/files/docker-trampoline

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+#
+# Ability to execute arbituary docker commands equals to root-shell, so this
+# script does some whitelisting.  The script should be launched with:
+#
+#   sudo --non-interactive /foo/bar/trampoline.sh \
+#         do-something-awesome.py \
+#         "{{ ds }}" \
+#         "{{ execution_date.isoformat() }}" \
+#         "{{ (execution_date + dag.schedule_interval).isoformat() }}" \
+#         other CLI arguments to script
+
+# FIXME: 1000 = `id -u benchmark` at the datacollector dom0
+uidno=1000
+
+script="$1"
+bucket="$2"
+isofrom="$3" # execution_date
+isotill="$4" # execution_date + schedule_interval
+shift 4
+
+(
+shopt -s extglob
+for var in "$bucket" "$isofrom" "$isotill"; do
+    # NB: `.` (dot) delimits microseconds in isoformat() but they are not welcome here
+    if [ -z "$var" -o -n "${var/+([-0-9T:])/}" ]; then
+        echo "$0: bad arg <$var> -> <${var/+([-0-9T:])/}>" 1>&2
+        exit 1
+    fi
+done
+) || exit 1
+
+docksafe="--rm --interactive --read-only"
+docknet="--network none"
+
+priv=/data/ooni/private
+pub=/data/ooni/public
```

Comment:
Maybe these need to be parametrised or if not at least mention in the comment in the header that this is the expected filesystem layout.